From 22b7d2a2bddf533904a308691fd86a75ddb28415 Mon Sep 17 00:00:00 2001
From: SheatNoisette <sheatnoisette@protonmail.com>
Date: Sun, 13 Aug 2023 18:24:54 +0200
Subject: [PATCH 5/5] drvgen: Rename list variables for py3

---
 tools/dct/data/EintData.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/tools/dct/data/EintData.py b/tools/dct/data/EintData.py
index 3b5184f92..3da372071 100755
--- a/tools/dct/data/EintData.py
+++ b/tools/dct/data/EintData.py
@@ -74,19 +74,19 @@ class EintData:
         key = 'gpio%s' %(gpio_num)
 
         if key in list(EintData._mode_map.keys()):
-            list =  EintData._mode_map[key]
-            if mode_idx < len(list) and mode_idx >= 0:
-                return list[mode_idx]
+            list_data =  EintData._mode_map[key]
+            if mode_idx < len(list_data) and mode_idx >= 0:
+                return list_data[mode_idx]
 
         return None
 
     @staticmethod
     def set_modeMap(map):
         for (key, value) in list(map.items()):
-            list = []
+            list_data = []
             for item in value:
-                list.append(item[6:len(item)-1])
-            map[key] = list
+                list_data.append(item[6:len(item)-1])
+            map[key] = list_data
 
         EintData._mode_map = map
 
-- 
2.41.0

